packages/epics/src/governance/components/document-card.tsx

```diff
@@ -0,0 +1,89 @@
+import {
+  Skeleton,
+  Card,
+  CardHeader,
+  CardTitle,
+  CardContent,
+} from '@hypha-platform/ui';
+import { Image } from '@hypha-platform/ui';
+import { PersonLabel } from '../../people/components/person-label';
+import { type Creator } from '../../people/components/person-label';
+import { DocumentBadges, type BadgeItem } from './document-badges';
+
+interface Document {
+  id: number;
+  creatorId: number;
+  title: string;
+  description?: string;
+  slug: string;
+  createdAt: Date;
+  updatedAt: Date;
```

Comment:
please only declare what is actually used. That is the hole point of it. When you redeclare the same schema, it doesn't make sense.